openc3/lib/openc3/script/storage.rb

```diff
@@ -133,6 +133,7 @@ def _get_storage_file(path, scope: $openc3_scope)
       Net::HTTP.start(uri.host, uri.port) do |http|
         request = Net::HTTP::Get.new uri
         http.request request do |response|
+          response.value() # Raises an HTTP error if the response is not 2xx (success)
```

Comment:
Not sure what happened that required this change but without it we were returning an XML file with a string containing 404 not found. Previously we were raising an error which is what this line does and that stops the script when calling get_target_file